src/loaders/ObjectLoader.js

```diff
@@ -466,8 +466,21 @@ THREE.ObjectLoader.prototype = {
 
 				case 'PerspectiveCamera':
 
-					object = Object.assign(
-							new THREE.PerspectiveCamera(), data );
+					object = new THREE.PerspectiveCamera(
+							data.fov, data.aspect, data.near, data.far );
+
+					if ( data.focus !== undefined ) object.focus = data.focus;
+					if ( data.zoom !== undefined ) object.zoom = data.zoom;
+
+					if ( data.filmGauge !== undefined ) {
+
+						if ( data.view !== null )
+							object.view = Object.assign( {}, data.view );
```

Comment:
@bhouston 
> Exporters that are often partially implemented

Yes, your suggestion to split it out makes sense. Then we also have to check `undefined` in case of `view`, since it's not present in all versions of the file format.

I don't really like the amount of clutter it adds. Maybe we should have `THREE.assignOptionals`, so we can write:

```javascript
THREE.assignOptionals( object, {
    focus: data.focus,
    zoom: data.zoom,
   // [...]
} );
```

It would not touch the object for properties that are `undefined`, assign `null` and deep-copy non-circular object structures (IOW JSON input), like `.view` in this case.